password_security/models/res_users.py

```diff
@@ -205,3 +205,12 @@ def _set_encrypted_password(self, uid, pw):
 
         self.write({"password_history_ids": [(0, 0, {"password_crypt": pw})]})
         return res
+
+    def action_reset_password(self):
+        """Disallow password resets inside of Minimum Hours"""
+        if self.env.context.get("install_mode"):
+            return
+        if not self.env.context.get("create_user"):
+            users = self.filtered(lambda user: user.active)
```

Comment:
The check on active is redundant but I made it on purpose, so that in case an user is inactive, [this standard error](https://github.com/odoo/odoo/blob/991d1c430f2c9bb0d2b2d7ac8f75338eafd2aced/addons/auth_signup/models/res_users.py#L166-L167) will be always raised, instead of the eventual `_validate_pass_reset`.

I just added the SUPERUSER_ID check.